flutter/ios/Classes/SwiftSentryFlutterPlugin.swift

```diff
@@ -3,27 +3,257 @@ import Sentry
 import UIKit
 
 public class SwiftSentryFlutterPlugin: NSObject, FlutterPlugin {
-  public static func register(with registrar: FlutterPluginRegistrar) {
-    let channel = FlutterMethodChannel(name: "sentry_flutter", binaryMessenger: registrar.messenger())
-    let instance = SwiftSentryFlutterPlugin()
-    registrar.addMethodCallDelegate(instance, channel: channel)
-  }
-
-  public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
-    result("iOS " + UIDevice.current.systemVersion)
-  }
-
-  override init() {
-    if let path = Bundle.main.path(forResource: "Info", ofType: "plist") {
-      if let resource = NSDictionary(contentsOfFile: path) {
-        if let dsn = resource.object(forKey: "SentryDsn") {
-          SentrySDK.start { options in
-              options.dsn = dsn as? String
-              options.debug = resource["SentryDebug"] as? Bool ?? false
-              options.attachStacktrace = true
-          }
-        }
-      }
-    }
-  }
+
+    var sentryOptions: Options?
+
+    public static func register(with registrar: FlutterPluginRegistrar) {
+        let channel = FlutterMethodChannel(name: "sentry_flutter", binaryMessenger: registrar.messenger())
+        let instance = SwiftSentryFlutterPlugin()
+        registrar.addMethodCallDelegate(instance, channel: channel)
+    }
+
+    public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+        switch call.method as String{
+            case "deviceInfos" :
+              deviceInfos(result: result)
+              break
+
+            case "initNativeSdk" :
+              initNativeSdk(call, result: result)
+              break
+
+        case "captureEnvelope" :
+            captureEnvelope(call, result: result)
+            break
+        default:
+            result(FlutterMethodNotImplemented)
+        }
+
+
+    }
+
+    private func deviceInfos(result: @escaping FlutterResult){
```

Comment:
I'd call this loadContexts or something, as I said, this is not only about the device info

Reply:
✅

Reply:
method is still called `deviceInfos`